Parse/cloud/main.js

```diff
@@ -41,6 +59,56 @@ Parse.Cloud.beforeSave("VentAudience", async (request) => {
 
 });
 
+Parse.Cloud.beforeSave("GroupDetails", async (request) => {
+  var currentGroup = request.object;
+  var groupName = await request.object.get("groupName");
+
+  request.object.set(
+    "searchName",
+    groupName.toLowerCase()
+  );
+
+});
+
+Parse.Cloud.afterSave("GroupDetails", async (request) => {
+  const SearchItem = Parse.Object.extend("SearchItem");
+  const searchItem = new SearchItem();
+  var currentGroup = request.object;
+  var groupName = await request.object.get("groupName");
+
+  searchItem.save({
+    searchName: groupName.toLowerCase(),
+    group: currentGroup
+  })
+  .then((searchItem) => {
```

Comment:
same here, the .then seems unused, we should either remove it or at least log an error